calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-management-cluster-helm.mdx

```diff
@@ -266,7 +246,7 @@ To install a {{prodname}} [management](create-a-management-cluster-helm#value) c
 1. Export the service port number
 
   ```bash
-  export EXT_NODE_PORT=<your-load-balancer-external-node-port>
+  export EXT_NODE_PORT=<your-external-load-balancer-port>
```

Comment:
The variable name still uses the node port terminology (if this was intentional, feel free to ignore)

Reply:
Changing to `EXT_LB_PORT`